docs/anomaly-detection/QuickStart.md

```diff
@@ -0,0 +1,103 @@
+---
+sort: 1
+weight: 1
+title: VictoriaMetrics Anomaly Detection Quick Start
+menu:
+  docs:
+    parent: "anomaly-detection"
+    weight: 1
+    title: Quick Start
+aliases:
+- /anomaly-detection/QuickStart.html
+---
+
+# VictoriaMetrics Anomaly Detection Quick Start
+
+It is recommended to read [README](https://docs.victoriametrics.com/anomaly-detection/)
+and [Overview](https://docs.victoriametrics.com/anomaly-detection/overview.html)
+before you start working with `vmanomaly`.
+
+## How to install and run `vmanomaly`
+
+>`vmanomaly` is a part of VictoriaMetrics Enterprise version. You can get a license key [**here**](https://victoriametrics.com/products/enterprise/trial/index.html).
+
+There are the following options exist:
+
+- [To run Docker image](#docker-image)
+- [To run in Kubernetes with Helm charts](#helm-charts)
+
+
+### Docker image
+
+You can run `vmanomaly` in a Docker container. It is the easiest way to start using `vmanomaly`.
```

Comment:
Let's use better wording here:

The simplest and quickest method to deploy `vmanomaly` is through Docker. Below are the steps to get `vmanomaly` up and running inside a Docker container.

First, you can (optionally) tag the `vmanomaly` Docker image for convenience using the following command:

```sh
docker image tag victoriametrics/vmanomaly:v1.9.2 vmanomaly
```

Second, start the `vmanomaly` Docker container with a license file, use the command below.
Make sure to replace `[YOUR_NETWORK]`, `[YOUR_LICENSE_FILE_PATH]`, and `[YOUR_CONFIG_FILE_PATH]` with your specific details:
...

P.s. Neither `docker pull victoriametrics/vmanomaly` nor `docker pull victoriametrics/vmanomaly:latest` worked for me, only exact versions are supported right now, like
```sh
docker pull victoriametrics/vmanomaly:v1.9.2
```

Reply:
@fred-navruzov maybe we should make the  `latest` tag available then?
It is convenient for users sometimes not to specify the version and it should be a super fast quickstart

Reply:
sounds fair, at least for the guide :+1: 
made :latest to point to current latest stable (1.9.2)